nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-client-dto/src/main/java/org/apache/nifi/web/api/dto/ProcessorConfigDTO.java

```diff
@@ -44,6 +44,7 @@
     private String comments;
     private String customUiUrl;
     private Boolean lossTolerant;
+    private Boolean executionNodeRestricted;
```

Comment:
This is not something that is configurable, so I think we want to put this in the ProcessorDTO, not the ProcessorConfigDTO. This is where we store other 'flags' about a processor, such as restricted, deprecation, supportsBatching, etc, etc.

Reply:
Understood. Makes more sense now. I'll move it to `ProcessorDTO`. One question though, I added the setter in [DTOFactory.createProcessorDTO()](https://github.com/apache/nifi/blob/master/nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/api/dto/DtoFactory.java#L2749-L2786), it has to be added to the [DTOFactory.copy()](https://github.com/apache/nifi/blob/master/nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/api/dto/DtoFactory.java#L3721-L3744) as well, right? Where and how the copy() call is used?

Reply:
Yes, it will need to be added to DtoFactory.copy() as well. That method is used when you copy & paste a processor for example.